### ISSUE TITLE
Hello. I just wanted to know how to log in to the bot

### ISSUE THREAD

**Issue by SaadatAlam** [Jacrac04/DFM-Bot#49]
Please use [discussions](https://github.com/Jacrac04/DFM-Bot/discussions) for this
My school only gave me a username and no email.

**Comment by Pottle** [comment]
yeah it doesn't work anymore, there is a popup every time you open it.
